lib/Db/CardMapper.php

```diff
@@ -165,6 +165,14 @@ public function queryCardsByBoards(array $boardIds): IQueryBuilder {
 		return $qb;
 	}
 
+	public function findToDelete() {
+		// add buffer of 5 min
+		$timeLimit = time() - (60 * 5);
+		$sql = 'SELECT id, title, owner, archived, deleted_at, last_modified FROM `*PREFIX*deck_cards` ' .
```

Comment:
This should be rather using the query builder. You may have a look at the other methods of this mapper class on how that works. Let me know if you need any further help on that :)

Reply:
You are right, Julius, fixed now